.github/workflows/linting.yml

```
@@ -63,16 +63,34 @@ jobs:
   markdown-link-check:
     name: Markdown Links (modified files)
     runs-on: ubuntu-latest
+    env:
+      BASE_BRANCH: ${{ github.base_ref }}
+      FILE_EXTENSION: .md
     steps:
       - name: Check out the repository
         uses: actions/checkout@v2
-
+      - name: Extract markdown from CSV.yaml
+        run: |-
+          sudo snap install yq --channel=v4/stable && \
+          yq e '.spec.description, .spec.links' config/manifests/bases/submariner.clusterserviceversion.yaml > bundle.md
```

```suggestion
          yq e '.spec.description, .spec.links' config/manifests/bases/submariner.clusterserviceversion.yaml > extracted_yaml_links.md
```

That `bundle.md` name is not super clear to me, maybe something more verbose would be quicker to understand.